LonaStudio/Utils/AttributedFont.swift

```diff
@@ -39,14 +39,22 @@ public class AttributedFont {
         self.lineBreakMode = lineBreakMode
     }
     
-    public var nsFont: NSFont {
-        if let targetFont = NSFontManager.shared().font(withFamily: fontFamily, traits: NSFontTraitMask(rawValue: 0), weight: weight.rawValue, size: fontSize) {
-            return targetFont
+    public func nsFont(_ fontSize: CGFloat) -> NSFont {
```

Comment:
Since `fontSize` is a member variable of `AttributedFont`, I don't think we should allow overriding it here. The `AttributedFont` instance should be updated before calling `nsFont`.

E.g. in the caller:

```swift
let attributedFont = /* some font */

attributedFont.fontSize = 14

let whatever = attributedFont.nsFont
```